ci/roundtrip_local_to_c8y.py

```diff
@@ -224,8 +224,20 @@ def assert_values(
 
     expected = list(range(0, int(publish_amount)))
 
-    print("Retrieved:", values)
-    print("Expected:", expected)
+    print("Retrieved values:")
+
+    for value in range(len(values)):
+
+        if value >= 1:
+            if (values[value - 1] + 1) != values[value]:
+                print("error!")
+
+        print(f"{values[value]:5} ", end="")
+        if int(values[value] + 1) % 20 == 0:  # use a new line when data is
+            print("")
```

Comment:
```suggestion
    for index, value in enumerate(values):

        if index >= 1: # I'm not sure what we are trying to achieve here?
            if (values[index- 1] + 1) != value:
                print("error!")

        print(f"{value:5} ", end="")
        try: 
            if int(values[value] + 1) % 20 == 0:  # use a new line when data is
                print("")
        except ValueError:
            print("do what you would do in case of an error")
```